regression/contracts/invar_assigns_alias_analysis/test.desc

```diff
@@ -0,0 +1,14 @@
+KNOWNBUG
```

Comment:
If it is marked as a known bug then the test will not be run.  Is it possible to either have just a test for the parsing or test for the "incorrect" results until it is implemented.

Reply:
@SaswatPadhi @aalok-thakkar

Reply:
There is one such test already, to check just the parsing. Please see: regression/contracts/invar_assigns_opt/main.c.

Reply:
Thanks.